portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -0,0 +1,104 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.utils.CommentDataStore;
+import com.google.sps.utils.ServletUtils;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.List;
+
+
+/** Servlet that handles posting comment content. */
+@WebServlet("/list-comment")
+public class ListCommentServlet extends HttpServlet {
+
+    private static final String PARAM_NAME_QUANTITY = "quantity";
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // Gets the displayed comment limit
+        int limit = getMaxNumberOfComments(request, response);
+
+        // Loads the comment from Datastore
+        List<Comment> comments = getComments(limit);
+
+        // Converts into json form
+        String json = convertToJsonUsingGson(comments);
+
+        // Sends the JSON as the response
+        sendJsonResponse(response, json);
+    }
+
+    private int getMaxNumberOfComments(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String maxNumOfCommentStr = ServletUtils.getParameter(
+                request, /*name=*/PARAM_NAME_QUANTITY, /*defaultValue=*/"10");
```

Comment:
No need /* name */ any more since you're passing in a named variable.

Reply:
Fixed